cmd/image-builder/main_test.go

```diff
@@ -707,6 +708,38 @@ type: qcow2
 arch: x86_64`)
 }
 
+func TestDescribeImageMinimal(t *testing.T) {
+	restore := main.MockNewRepoRegistry(testrepos.New)
+	defer restore()
+
+	restore = main.MockDistroGetHostDistroName(func() (string, error) {
+		return "centos-9", nil
+	})
+	defer restore()
+
+	restore = main.MockCurrentArch(func() arch.Arch {
+		return arch.ARCH_X86_64
+	})
+	defer restore()
+
+	restore = main.MockOsArgs([]string{
+		"describe",
+		"qcow2",
+	})
+	defer restore()
+
+	var fakeStdout bytes.Buffer
+	restore = main.MockOsStdout(&fakeStdout)
+	defer restore()
+
+	err := main.Run()
+	assert.NoError(t, err)
+
+	assert.Contains(t, fakeStdout.String(), `distro: centos-9
+type: qcow2
+arch: x86_64`)
```

Comment:
I wonder if we could avoid this mocking If we just use `fmt.Sprintf("type: qcow2\narch: %s", arch.Current())` here? Its not a big deal and maybe having the ability is useful in more contexts. But it less code and all that :)

Reply:
done